### ISSUE TITLE
Missing Java type information causes camel-main build to fail

### ISSUE THREAD

**Issue by orpiske** [apache/camel-k-runtime#677]
The build seems unable to resolve `javaType` from the [QuteComponent ](https://github.com/apache/camel-quarkus/blob/19792dc1aba16aaedce788fd1aff43da4c1c2924/extensions/qute/component/src/generated/resources/org/apache/camel/component/qute/qute.json#L10) which seems to cause it to fail to generate the catalog. This fails the build.

**Comment by orpiske** [comment]
Log: 

```
 Error: [ERROR] Failed to execute goal org.apache.camel.k:camel-k-maven-plugin:1.9.0-SNAPSHOT:generate-catalog (generate-catalog) on project camel-k-catalog-generator: Execution generate-catalog of goal org.apache.camel.k:camel-k-maven-plugin:1.9.0-SNAPSHOT:generate-catalog failed: javaTypes element -> [Help 1]
Error: [ERROR] 
Error: [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
Error: [ERROR] Re-run Maven using the -X switch to enable full debug logging.
Error: [ERROR] 
Error: [ERROR] For more information about the errors and possible solutions, please read the following articles:
Error: [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

**Comment by orpiske** [comment]
Fixed with #721 and apache/camel-k#2447
